src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -23,69 +21,118 @@
+                : emailConfig.getSubmittedDissolutionTeamEmailType();
+
+        return new EmailContent.Builder()
+                .withOriginatingAppId(emailConfig.getOriginatingAppId())
+                .withCreatedAt(dateTimeSupplier.get())
+                .withMessageType(typeOfEmail)
+                .withMessageId(UUID.randomUUID().toString())
+                .withEmailAddress(emailAddress)
+                .withData(data)
+                .build();
+    }
+
+    private Map<String, Object> constructEmailDataMap(String companyName, Objection objection, String email) {
         Map<String, Object> data = new HashMap<>();
 
         LocalDate submittedOn = objection.getCreatedOn().toLocalDate();
 
-        String subject = emailSubject.replace("{{ COMPANY_NUMBER }}", companyNumber);
+        String emailSubject = emailConfig.getEmailSubject();
+        String subject = emailSubject.replace("{{ COMPANY_NUMBER }}", objection.getCompanyNumber());
         data.put("subject", subject);
-        data.put("to", emailAddress);
         data.put("date", FormatUtils.formatDate(submittedOn));
         data.put("objection_id", objection.getId());
+        data.put("to", email);
         data.put("company_name", companyName);
-        data.put("company_number", companyNumber);
+        data.put("company_number", objection.getCompanyNumber());
+        data.put("objection_id", objection.getId());
         data.put("reason", objection.getReason());
         data.put("attachments", objection.getAttachments());
+        return data;
+    }
 
-        EmailContent emailContent = new EmailContent.Builder()
-                .withOriginatingAppId(originatingAppId)
-                .withCreatedAt(dateTimeSupplier.get())
-                .withMessageType(submittedCustomerEmailType)
-                .withMessageId(UUID.randomUUID().toString())
-                .withEmailAddress(emailAddress)
-                .withData(data)
-                .build();
-
-        logger.debugContext(requestId, "Calling Kafka client to send email");
-        kafkaEmailClient.sendEmailToKafka(emailContent);
-        logger.debugContext(requestId, "Successfully called Kafka client");
+    protected String[] getDissolutionTeamRecipients(String jurisdiction) {
+        switch(jurisdiction) {
+            case "england":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
+            case "wales":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
+            case "england-wales":
+                return emailConfig.getEmailRecipientsCardiff().split(",");
+            case "scotland":
+                return emailConfig.getEmailRecipientsEdinburgh().split(",");
+            case "northern-ireland":
+                return emailConfig.getEmailRecipientsBelfast().split(",");
+            default:
+                return emailConfig.getEmailRecipientsCardiff().split(",");
```

Comment:
Are we sure this isn't a (runtime exception) error scenario, if jurisdiction doesn't match what we expect?

Reply:
No we dealt with this before - Cardiff is the default.

Reply:
Yeh I know - I had a comment above about this saying the opposite - I think there are multiple points of view here.

Reply:
Ok. Something like that always worth a comment in the code I think, but won't hold up my approval.